### ISSUE TITLE
Default Building Settings - Missing Mod Menu?

### ISSUE THREAD

**Issue by Ratione** [AzeTheGreat/ONI-Mods#15]
Hey, so I started playing again recently, after the merge-down broke everything. 

After updating the default building setttings mod again, I decided I wanted to change one of the mods settings, and I discovered the mod menu was missing. 

I used mod menu to force an update a few times, and uninstalled/reinstalled it a few times, but the mod menu is still missing

**Comment by AzeTheGreat** [comment]
Could you please provide the log file so that I can confirm the mod version that the game is loading with?

**Comment by Ratione** [comment]
Yeah, so something in my install broke. No idea what. At some point I ended up doing... something, that fully bricked my install. I got stuck in this weird launch loop that I cannot begin to explain. I tried reinstalling several times, to no avail. Eventually, I had to uninstall the game, clear the documents folder, and basically wipe everything relating to the game from my computer, going so far as to clear the steam download cache, THEN reinstall, and only then would the game work properly.

Upon returning to functionality, your mod's menu showed up, as usual, and appears to be functioning. 



Honestly, I have no fuckin clue what actually went wrong, but it appears to be working now. I suspect that it was some bizarre interaction with the mod updater I was using, but I have basically no way to test this, so i am going to close this here, and chalk it up to some weird government conspiracy. Or something.
